infra/base-images/base-runner/coverage

```diff
@@ -227,7 +235,8 @@ wait
 
 if [[ $FUZZING_LANGUAGE == "go" ]]; then
   $SYSGOPATH/bin/gocovmerge $DUMPS_DIR/*.profdata > fuzz.cov
-  go tool cover -html=fuzz.cov -o $REPORT_ROOT_DIR/index.html
+  # TODO: Once oss-fuzz upgrades go to 1.18, use "go" instead of "gotip":
```

Comment:
Can you please put your handle here so we know who should do this from looking at the code?